browserscripts/browser/collectStats.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  requires: { privilege: true },
+  function: async function() {
+    let stats;
+    /*
+		 * Collect all the performance statistics generated
+		 * during the execution.
+		 */
+    await ChromeUtils.collectPerfStats().then(value => {
```

Comment:
Everything is async here already, so there's no need to `await` and `then`.  I expect:
```
const stats = await ChromeUtils.collectPerfStats();
return JSON.parse(stats);
```
to be fine.